packages/react-query/src/index.ts

```diff
@@ -1,5 +1,31 @@
+import { AnyProcedure, inferProcedureInput } from '@trpc/server';
+import { QueryType, getArrayQueryKey } from './internals/getArrayQueryKey';
+import { getQueryKey as $getQueryKey } from './internals/getQueryKey';
+import { DecorateProcedure } from './shared';
+
 export * from '@trpc/client';
 
 export { createTRPCReact, type CreateTRPCReact } from './createTRPCReact';
 export { createReactQueryHooks } from './interop';
 export type { inferReactQueryProcedureOptions } from './utils/inferReactQueryProcedure';
+
+export function getQueryKey<
```

Comment:
Shouldn't we add back the comment from the previous `getQueryKey`?

```suggestion
/**
 * Method to extract the query key for a procedure
 * @param type - defaults to `any`
 * @link https://trpc.io/docs/useContext#-the-function-i-want-isnt-here
 */
export function getQueryKey<
```